Assets/Scripts/Isamu/Map/Navigation/AffordableNodesRequest.cs

```diff
@@ -0,0 +1,16 @@
+namespace Isamu.Map.Navigation
+{
+    public class AffordableNodesRequest
```

Comment:
Bringing this into its own file, since we'll probably need it for determining attack ranges as well.